src/app/views/sidebar/resource-explorer/panels/PathsReview.tsx

```diff
@@ -67,6 +67,10 @@ const PathsReview = (props: IPathsReview) => {
     setSelectedItems(content);
   };
 
+  const panelOverlayProps: IOverlayProps = {
+    isDarkThemed: true
```

Comment:
Would it make sense to have this take the true/false value from the current theme?

Reply:
Yes, we can do that. Let me make the change